vigilate_backend/views.py

```diff
@@ -258,16 +260,17 @@ def get_queryset(self):
         else:
             return Station.objects.filter(user=self.request.user.id)
 
-class SessionViewSet(viewsets.ModelViewSet):
+class SessionViewSet(viewsets.mixins.CreateModelMixin,
+                     viewsets.mixins.DestroyModelMixin,
+                     viewsets.mixins.ListModelMixin,
+                     viewsets.GenericViewSet):
     """View for session
     """
     serializer_class = SessionSerializer
-    http_method_names = ['get', 'post', 'delete']
```

Comment:
Two slight problems:
- OPTIONS request are not allowed. We could just add it in the list, but there is still the next "problem".
- the "get" allow both list() and retrieve(). But we don't want retrieve() to be present.